torch/csrc/jit/passes/onnx/helper.cpp

```diff
@@ -48,5 +49,40 @@ void buildParamsMapFromValueToParamsMap(
     paramsDict.insert(nameTensorParamPair.second);
   }
 }
+
+c10::optional<at::ScalarType> ONNXTypeToATenType(int32_t onnx_type) {
```

Comment:
Just curious if we have other places in the code where this map is needed, and if this is one centralized location for it or do we have this in multiple places?

Reply:
yes, there are multiple places such onnx type <-> torch type mapping is needed. The aim is to have this as a centralized location. 
This also helps with clang-tidy, as clang-tidy cannot run on any file that includes <onnx_pb.h>, a generated header. Reducing the number of files depending on <onnx_pb.h> solely for getting `ONNX::TensorProto_DataType` definition.